runelite-client/src/main/java/net/runelite/client/plugins/kourendlibrary/KourendLibraryPanel.java

```diff
@@ -53,7 +66,10 @@ void init()
 		GroupLayout layout = new GroupLayout(this);
 		setLayout(layout);
 
+		setBackground(ColorScheme.DARK_GRAY_COLOR);
```

Comment:
I am starting to feel like that all these `setBackground(ColorScheme.DARK_GRAY_COLOR)` should be instead in PluginPanel

Reply:
They are, however setOpaque seems to not work for some jpanels, leaving them with a different background than the PluginPanel, but for that example in specific, I think you're right.